core/admin/migrations/versions/0ba45693748d_.py

```diff
@@ -0,0 +1,22 @@
+"""Add user.change_pw_next_login
+
+Revision ID: 0ba45693748d
+Revises: 6b8f5e8caaa9
+Create Date: 2023-08-10 09:20:50.458092
+
+"""
+
+# revision identifiers, used by Alembic.
+revision = '0ba45693748d'
+down_revision = '6b8f5e8caaa9'
+
+from alembic import op
+import sqlalchemy as sa
+
+
+def upgrade():
+    op.add_column('user', sa.Column('change_pw_next_login', sa.Boolean(), nullable=False, server_default=sa.sql.expression.false()))
+
+
+def downgrade():
+    pass
```

Comment:
Why not remove the column here?

Reply:
Other migrations in Mailu don't. I guess the rationale is that we don't want to loose data.

Do you want me to implement it here?

Reply:
Done in a5f3c7ea